File: GamesServer/GamesServer/DTO/PlayerChooseCardData.cs
namespace GamesServer.DTO
{
    public class PlayerChooseCardData
    {
        public int PlayerId { get; set; }
        public int CardId { get; set; }

        public PlayerChooseCardData(
            int playerId,
            int cardId)
        {
            PlayerId = playerId;
            CardId = cardId;
        }
    }
}

File: GamesServer/PhytonServer/Scripts/DTO/PlayerActions/PlayerChooseCardData.py



class PlayerChooseCardData:
    def __init__(self, player_id,card_id):
        self.player_id = player_id
        self.card_id = card_id

    def to_dict(self):
        return {
            "PlayerId": self.player_id,
            "CardId": self.card_id
        }